README.md

```diff
@@ -114,6 +114,8 @@ cd build
 ccmake ../
 make
 ```
+Note: You can configure the ccmake environment if you know you will use some module (put them in "ON"). For exemple you may need some tools presented here: #matlab-software.
```

Comment:
Some minor comments: avoid the `Note:` part, `exemple` ---> `example`.
You can link a subsection using the `[text of the link](#matlab-software)` syntax, play with the "preview" button to see the effect of your modification, thanks!